jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/core/beanvalidation/BeanValidationDiagnosticsCollector.java

```diff
@@ -1,354 +1,281 @@
+import static org.eclipse.lsp4jakarta.jdt.core.beanvalidation.BeanValidationConstants.BOOLEAN;
+import static org.eclipse.lsp4jakarta.jdt.core.beanvalidation.BeanValidationConstants.BYTE;
+import static org.eclipse.lsp4jakarta.jdt.core.beanvalidation.BeanValidationConstants.CHAR_SEQUENCE;
+import static org.eclipse.lsp4jakarta.jdt.core.beanvalidation.BeanValidationConstants.DECIMAL_MAX;
+import static org.eclipse.lsp4jakarta.jdt.core.beanvalidation.BeanValidationConstants.DECIMAL_MIN;
+import static org.eclipse.lsp4jakarta.jdt.core.beanvalidation.BeanValidationConstants.DIAGNOSTIC_CODE_INVALID_TYPE;
+import static org.eclipse.lsp4jakarta.jdt.core.beanvalidation.BeanValidationConstants.DIAGNOSTIC_CODE_STATIC;
+import static org.eclipse.lsp4jakarta.jdt.core.beanvalidation.BeanValidationConstants.DIAGNOSTIC_SOURCE;
+import static org.eclipse.lsp4jakarta.jdt.core.beanvalidation.BeanValidationConstants.DIGITS;
+import static org.eclipse.lsp4jakarta.jdt.core.beanvalidation.BeanValidationConstants.DOUBLE;
+import static org.eclipse.lsp4jakarta.jdt.core.beanvalidation.BeanValidationConstants.EMAIL;
+import static org.eclipse.lsp4jakarta.jdt.core.beanvalidation.BeanValidationConstants.FLOAT;
+import static org.eclipse.lsp4jakarta.jdt.core.beanvalidation.BeanValidationConstants.FUTURE;
+import static org.eclipse.lsp4jakarta.jdt.core.beanvalidation.BeanValidationConstants.FUTURE_OR_PRESENT;
+import static org.eclipse.lsp4jakarta.jdt.core.beanvalidation.BeanValidationConstants.INTEGER;
+import static org.eclipse.lsp4jakarta.jdt.core.beanvalidation.BeanValidationConstants.LONG;
+import static org.eclipse.lsp4jakarta.jdt.core.beanvalidation.BeanValidationConstants.MAX;
+import static org.eclipse.lsp4jakarta.jdt.core.beanvalidation.BeanValidationConstants.MIN;
+import static org.eclipse.lsp4jakarta.jdt.core.beanvalidation.BeanValidationConstants.NEGATIVE;
+import static org.eclipse.lsp4jakarta.jdt.core.beanvalidation.BeanValidationConstants.NEGATIVE_OR_ZERO;
+import static org.eclipse.lsp4jakarta.jdt.core.beanvalidation.BeanValidationConstants.NOT_BLANK;
+import static org.eclipse.lsp4jakarta.jdt.core.beanvalidation.BeanValidationConstants.PAST;
+import static org.eclipse.lsp4jakarta.jdt.core.beanvalidation.BeanValidationConstants.PAST_OR_PRESENT;
+import static org.eclipse.lsp4jakarta.jdt.core.beanvalidation.BeanValidationConstants.PATTERN;
+import static org.eclipse.lsp4jakarta.jdt.core.beanvalidation.BeanValidationConstants.POSITIVE;
+import static org.eclipse.lsp4jakarta.jdt.core.beanvalidation.BeanValidationConstants.POSTIVE_OR_ZERO;
+import static org.eclipse.lsp4jakarta.jdt.core.beanvalidation.BeanValidationConstants.SET_OF_ANNOTATIONS;
+import static org.eclipse.lsp4jakarta.jdt.core.beanvalidation.BeanValidationConstants.SET_OF_DATE_TYPES;
+import static org.eclipse.lsp4jakarta.jdt.core.beanvalidation.BeanValidationConstants.SHORT;
+import static org.eclipse.lsp4jakarta.jdt.core.beanvalidation.BeanValidationConstants.STRING;

```

Comment:
(Optional) suggestion to simplify these import statements

Reply:
I guess those are for trying to simplify constants usage, unless you want to qualified each of them to get rid of them all or use the wildcard?

Reply:
Right I was thinking wildcard

Reply:
I can double check the constants, if they are already most of them here, it will be fine to pull few more in by using wildcard.

Reply:
About 65%